src/option-parser.cc

```diff
@@ -86,97 +172,115 @@ void parse_options(OptionParser* parser, int argc, char** argv) {
         }
 
         if (best_count > 1) {
-          error(parser, "ambiguous option \"%s\"", arg);
+          Errorf("ambiguous option \"%s\"", arg);
           continue;
         } else if (best_count == 0) {
-          error(parser, "unknown option \"%s\"", arg);
+          Errorf("unknown option \"%s\"", arg);
```

Comment:
Just single quotes here to avoid escaping?  This also happens to match how gnu tools format their option errors:

```ls --dsflgkj
ls: unrecognized option '--dsflgkj'
Try 'ls --help' for more information.
```